File: tournaments/lateRide/lateRide.js
function lateRide(n) {
    const minutes = Math.floor(n / 60);
    const seconds = n - (Math.floor(n / 60) * 60);

    return Math.floor(minutes / 10) + (minutes % 10) + (seconds % 10) + Math.floor(seconds / 10);
}

File: tournaments/lateRide/lateRide.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub lateRide {
    my ($n) = @_;

    my $minutes = int($n / 60);
    my $seconds = $n - (int($n / 60) * 60);

    return int($minutes / 10) + ($minutes % 10) + int($seconds / 10) + ($seconds % 10);
}
